scripts/hpo_codesystem.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import json
+import os
+import sys
+
+# OWL to FHIR transformer compiled from https://github.com/aehrc/fhir-owl
+FHIR_OWL_BIN = "fhir-owl-1.1.0-SNAPSHOT.jar"
+
+# Convert OWL file to FHIR CodeSystem
+ret = os.system(f'wget -N http://purl.obolibrary.org/obo/hp.owl && java -jar {FHIR_OWL_BIN} -i hp.owl -o hpo_codesystem.json -id hpo -name "Human Phenotype Ontology" -content not-present -mainNs http://purl.obolibrary.org/obo/HP_ -descriptionProp http://purl.org/dc/elements/1.1/subject -status active -codeReplace _,:')
```

Comment:
```
ret = os.system(
    f'''
    wget -N http://purl.obolibrary.org/obo/{terminology}.owl && 
    java -jar {FHIR_OWL_BIN} -i {terminology}.owl -o codesystem_{terminology}.json -codeReplace _,:
    '''
)
```

Reply:
This modification would fail if you type hpo instead of hp. I think we need to provide a few safe aliases.

Reply:
Oh, it was hp instead of hpo! Totally agreed!